tests/e2e/local/kind/setup_host.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Delete any previous kind cluster
+kind delete
+
+echo "Starting Kind."
+
+# Start kind
+kind create
+
+# Set KubeConfig
+mkdir -p "$HOME/.kube"
+sudo mv "$HOME/.kube/kind-config-1" "$HOME/.kube/config"
```

Comment:
can we make sure the file exists before moving it? in the future I think probably we should use the apimachinery to write the context details to the standard `$HOME/.kube/config` instead of a seperate file. I haven't had a chance to do this yet though.